mod_reforged/hooks/config/strings.nut

```diff
@@ -863,6 +864,15 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
 	}),
+	RF_CalculatedStrikes = ::UPD.getDescription({
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Damage is increased by " + ::MSU.Text.colorGreen("20%") + " against anyone who\'s turn has not started yet."
```

Comment:
- `whose`
- Please add hyperlink for `[turn|Concept.Turn]`.
- `not started yet in the current [round|Concept.Round].` to make it clear that it is a per-round thing and not a once per combat thing.